platforms/android/library/src/main/java/io/element/android/wysiwyg/internal/viewmodel/EditorViewModel.kt

```diff
@@ -256,6 +256,6 @@ internal class EditorViewModel(
     }
 
     private fun stringToSpans(string: String): CharSequence =
-        htmlConverter.fromHtmlToSpans(string)
+        htmlConverter?.fromHtmlToSpans(string) ?: ""
```

Comment:
nit: Should we log an error if htmlConverter is null here?

Reply:
Log or actually throw? The only reason why `HtmlConverter` could be null here is if there's been some configuration issue, so it seems reasonable to me to crash, but maybe we don't want to be so strict?

Reply:
Personally I'd prefer to er on the side of caution and not crash, in case we accidentally introduce a bug that triggers it in future.